Nodejs/Product/Nodejs/Repl/NodejsReplEvaluator.cs

```diff
@@ -39,6 +39,20 @@ public NodejsReplEvaluator(INodejsReplSite site)
             this._site = site;
         }
 
+        private string nodeExePath;
+
+        public string NodeExePath
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(this.nodeExePath))
+                {
+                    this.nodeExePath = this.GetNodeExePath();
```

Comment:
Is any thread safety required here?

Reply:
no, initialization is all on the UI thread.